feature-libs/asm/customer-360/components/asm-customer-table/asm-customer-table.model.ts

```diff
@@ -17,3 +17,14 @@ export interface CustomerTableColumn {
 export interface TableEntry {
   [key: string]: string | number | undefined;
 }
+
+export interface GeneralEntry extends TableEntry {
```

Comment:
Moved this here because we no longer use the GeneralEntry interface in ActivityList and I didn't want to break the ams customer table unit tests